lstchain/reco/utils.py

```diff
@@ -555,7 +564,7 @@ def filter_events(events,
                 if v > 0:
                     log.warning(f'{k} : {v}')
 
-    return events[filter]
+    return events[filter.to_numpy()]
```

Comment:
This `to_numpy` should not be needed, right? Since `filter` is a numpy array? Doesn't this even fail?

Reply:
The filter is, but the mask will not be for DataFrame. It gives `pandas.Series`, but with `to_numpy` it works for both

Reply:
I don't understand, since there is only the `filter` variable here, and no `mask` variable.

Reply:
Ah, the problem is that `array &= series` will actually result in `array` becoming a `Series`.

To prevent that, use `array[:] &= series`

Reply:
Sorry, I was incorrect earlier. for DataFrame, `filter` is `Series` and so is the corresponding `mask`. But, for Tables, `filter` and `mask` are `array`.
So, to have a common method, I used `to_numpy()` in the return